test/llvm-pass-dump-parser-tests.js

```diff
@@ -0,0 +1,70 @@
+import * as properties from '../lib/properties';
+
+import {LlvmPassDumpParser} from '../lib/parsers/llvm-pass-dump-parser';
+
+const languages = {
+    'c++': {id: 'c++'},
+};
+
+function deepCopy(obj) {
+    return JSON.parse(JSON.stringify(obj));
```

Comment:
this is not a good deep copy function but it works for the objects in question here